lib/models/subscription.js

```diff
@@ -119,6 +120,17 @@ module.exports = class Subscription extends Sequelize.Model {
     return results
   }
 
+  static async agedSyncCounts () {
+    const ageThreshold = moment().subtract(7, 'd').format('YYYY-MM-DD')
```

Comment:
I just chose 7 days as a threshold for aged jobs. We can definitely tweak this.

Reply:
If we change the threshold, will the metric reported be invalid for previous times? Do we need to add the threshold to the metric name so that it doesn't suddenly shift the dashboards/monitors in an unexpected way?